tests/scripts/thread-cert/node.py

```diff
@@ -1623,6 +1623,12 @@ def router_table(self):
 
         return router_table
 
+    def link_metrics_query_single_probe(self, dst_addr, typeid_flags):
```

Comment:
nit: specify the type for arguments?

Reply:
Do you mean to add comments here?

Reply:
use python typing 
FYI: https://docs.python.org/3/library/typing.html